### ISSUE TITLE
Change Informed Consent Wording

### ISSUE THREAD

**Issue by markwilliamfirth** [dwyl/best-evidence#149]
Change "encrypted database" to "secure database"

**Comment by SimonLab** [comment]
![image](https://user-images.githubusercontent.com/6057298/29514317-ff243794-865f-11e7-9c29-e029338d5672.png)

**Comment by ajburls** [comment]
@SimonLab @markwilliamfirth This wording is better. Thanks. We discussed that the reminder information below the I agree should be put above the I agree.

I noticed a bug when testing this and that is that the "register new account" link on the login page just takes you to the search page if you have logged in before and have cookies. So this either needs to be changed to log out or direct to correct page.

**Comment by SimonLab** [comment]
Thanks for testing and to remind us that the reminder should be put above the the "I agree" button.

Can I ask a bit more details on the bug you notice. Normally if the user is currently logged in, she shouldn't be able to access the "login", "consent" and "create user page" so she shouldn't be able in the first place able to click on the "register new account" link. Do you know how you manage to access the login page by being logged in? I will double check on my side.

**Comment by ajburls** [comment]
@SimonLab I have just tried to reproduce this but without success. All I can tell you is that I opened the app, it went to the login screen (COmputer filled in my email and password in respective fields) but when I pressed register a new account, it went to search. Now it is just going straight to search and I don't get to the login field. (Perhaps in between a new update?)

I will try and get screenshots if it happens again.
